From e333083bf36f347ddd14d3e91a0f3c2e9038180b Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Tue, 12 Sep 2017 15:45:05 -0500
Subject: [PATCH 097/256] ARM: davinci: Update da8xxevm Maintainers

The e-mail addresses for DA8XXEVM BOARD and DA850_AM18XXEVM BOARD
are invalid.

Remove DA8XXEVM.

Update DA850_AM18XXEVM to have me be the maintainer since I work
for Logic PD and have access to OMAP-L138 and AM1808 EVM kits.

Signed-off-by: Adam Ford <aford173@gmail.com>
Reviewed-by: Tom Rini <trini@konsulko.com>
---
 board/davinci/da8xxevm/MAINTAINERS | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/board/davinci/da8xxevm/MAINTAINERS b/board/davinci/da8xxevm/MAINTAINERS
index f32ce6633a..99b4786dd5 100644
--- a/board/davinci/da8xxevm/MAINTAINERS
+++ b/board/davinci/da8xxevm/MAINTAINERS
@@ -1,13 +1,7 @@
-DA8XXEVM BOARD
-M:	Nick Thompson <nick.thompson@gefanuc.com>
-S:	Maintained
-F:	board/davinci/da8xxevm/
-F:	include/configs/da830evm.h
-F:	configs/da830evm_defconfig
-
 DA850_AM18XXEVM BOARD
-M:	Sudhakar Rajashekhara <sudhakar.raj@ti.com>
+M:	Adam Ford <aford173@gmail.com>
 S:	Maintained
+F:	board/davinci/da8xxevm/
 F:	include/configs/da850evm.h
 F:	configs/da850_am18xxevm_defconfig
 F:	configs/da850evm_defconfig
-- 
2.13.6

